.github/workflows/push.yaml

```diff
@@ -0,0 +1,60 @@
+name: Postsubmit Push
+
+on:
+
+  push:
+    branches:
+    - master
+
+jobs:
+
+    push:
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v1
+
+      - name: Mount bazel cache
+        uses: actions/cache@v1
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel
+
+      - name: Install bazelisk
+        run: |
+          curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.6.1/bazelisk-linux-amd64"
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          mv bazelisk-linux-amd64 "${GITHUB_WORKSPACE}/bin/bazel"
+          chmod +x "${GITHUB_WORKSPACE}/bin/bazel"
+
+      - name: Build
+        run: |
+          set -eux
+
+          SHA=$(git rev-parse HEAD)
```

Comment:
It uses some env  var from github action, so I will keep it inline for now since the logic is still quite simple for now. We can extract it out to a dedicated script when it becomes complex.